### DISCUSSION TITLE
Directly use mp3 format in speaker classification?

### DISCUSSION THREAD

**Discussion by praveenmathew93** [speechbrain/speechbrain#1887] [Q&A]
I am training a speaker classification model with Speechbrain on the commonvoice data.

My qution is, does speechbrain accomodate mp3 audio samples directly?
Or should I always convert them to .wav or .flac before I send them for training like I do now? 

Apoligies if this has been discussed before and I have missed it out. 

Thanks!

**Comment by TParcollet** [comment]
Hi, SB uses torchaudio. Torchaudio supports MP3 via FFMPEG. So yes, you can do it (it will be slow from my experience).

**Reply by asumagic** [reply]
Seems to me that the `sox_io` backend supports MP3 natively, so I don't think ffmpeg will actually be involved?
